README.md

```diff
@@ -0,0 +1 @@
+# Microverse-Trial-3

```

Comment:
I would advise you to use the README template suggested in the project requirements. Some of the sections in that template are mandatory.